docs/backend/configuration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Configuration
+---
+
+# Configuration
+
+We currently use two gems for configuring the application:
+
+- [rails-settings-cached](https://github.com/huacnlee/rails-settings-cached)
+- [ENVied](https://github.com/eval/envied)
```

Comment:
This is just a nitpicking but since we talk about Envied and rails-settings-cached in this order, maybe we should swap the order here :D

It's also the most important configuration so it's ok it's listed first

Reply:
Not nitpicking at all, I had changed their order in one place and not in the other, which also explains the reversed screenshots. Thanks for noticing 🙏